apps/src/craft/code-connection/levels.js

```diff
@@ -45,25 +45,23 @@ module.exports = {
           craftBlock('decoration') +
           craftBlock('getnameof') +
           craftBlock('getdataof') +
-          craftBlock('createblock', `<value name='BLOCKTYPE'>${block('text')}</value><value name='BLOCKDATA'>${block('text')}</value>`)) +
+          craftBlock('createblock')) +
       category('Blocks',
         craftBlock('tptotarget') +
         craftBlock('tptopos') +
         craftBlock('fill') +
         craftBlock('give') +
         craftBlock('kill') +
-        craftBlock('setblock', `<value name='ITEM'>${craftBlock('block')}</value>`) +
+        craftBlock('setblock') +
```

Comment:
I agree — my preference would be to always include the default arguments.  That's the way we've standardized for the rest of the Code Studio platform because it requires fewer mouse clicks.